lib/matplotlib/dviread.py

```diff
@@ -583,7 +583,7 @@ class Vf(Dvi):
 
     Parameters
     ----------
-    filename : string or bytestring
+    filename : str or bytes
```

Comment:
throughout the module: this should just be "path-like" no?

Reply:
Technically, yes. We have not used "path-like" anywhere so far and I'm not quite sure if less experienced users know what "path-like" is. As a compromise I've used "str or path-like" even though technically str is already a path-like.